### PR TITLE
Bump Ruby to 2.6.6 and use GitHub Actions instead of Jenkins

### PR DESCRIPTION
**Pull request by issyl0**
- This moves us ever closer towards all the easy migrations earmarked in [RFC 123](https://github.com/alphagov/govuk-rfcs/blob/master/rfc-123-github-actions-ci.md) being done. 😱
- This also adds RuboCop and the corresponding `lint` Rake task to the `bundle exec rake` defaults. And the version of Ruby was behind the latest.

https://trello.com/c/PzpcnUdS/144-migrate-all-skipdeploytointegration-repos-ci-to-github-actions

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -47,57 +47,55 @@
...
-  #     --seed 1234
-  config.order = :random
-
-  # Seed global randomization in this process using the `--seed` CLI option.
-  # Setting this allows you to use `--seed` to deterministically reproduce
-  # test failures related to randomization by passing the same `--seed` value
-  # as the one that triggered the failure.
-  Kernel.srand config.seed
-=end
+  # The settings below are suggested to provide a good initial experience
```
Comment (benthorner) [inline spec/spec_helper.rb]: The auto-correct has messed up these comments. We should just delete them.

File: .rubocop.yml [hunk 2]
```diff
@@ -1,3 +1,8 @@
 inherit_gem:
   rubocop-govuk:
     - config/default.yml
+
+Metrics/BlockLength:
```
Comment (benthorner) [inline .rubocop.yml]: Maybe this can wait for https://github.com/alphagov/rubocop-govuk/pull/65?

Comment (issyl0) [reply .rubocop.yml]: Cool. I await a new release!

**Comment by issyl0** [comment]
In accordance with the still draft but broadly sensible [Actions conventions doc](https://github.com/alphagov/govuk-developer-docs/pull/2533), I renamed this file `ci.yml` and added a name.

**Review by sihugh — APPROVED** [review]

**Pull request merged**